### PR TITLE
add testing slides; add live component example

### PR DESCRIPTION
**Pull request by DSchau**

### CODE REVIEW

File: src/slides/12-testing/16.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Protractor
+
+De-faco end to end testing library for Angular/AngularJS applications
```
Comment (mcacek) [inline src/slides/12-testing/16.md]: de-facto

**Review by mcacek — CHANGES_REQUESTED** [review]
Just the little typo.

File: src/slides/12-testing/16.md (hunk 1 above)
Comment (DSchau) [reply src/slides/12-testing/16.md]: `de-faco` Latin for "I f'ed up"

Comment (mcacek) [reply src/slides/12-testing/16.md]: hahah

**Review by mcacek — APPROVED** [review]

**Comment by mcacek** [comment]
🌮 🎉

**Pull request merged**